examples/experimental/i3s-17-and-debug/helpers/build-minimap-data.js

```diff
@@ -0,0 +1,25 @@
+import {Vector3} from '@math.gl/core';
+import {Ellipsoid} from '@math.gl/geospatial';
+import {OrientedBoundingBox} from '@math.gl/culling';
+
+export function buildMinimapData(tiles) {
+  return tiles
+    .map(tile => {
+      if (!tile.selected || !tile.viewportIds.includes('main')) {
+        return null;
+      }
+      const boundingVolume = tile.boundingVolume;
+      const cartographicOrigin = new Vector3();
+      Ellipsoid.WGS84.cartesianToCartographic(boundingVolume.center, cartographicOrigin);
+      let radius = boundingVolume.radius;
+      if (!radius && boundingVolume instanceof OrientedBoundingBox) {
```

Comment:
Should we check for `instanceof BoundingSphere` before accessing radius rather than mixing type checks like this?

Reply:
Unfortunately `instanceof BoundingSphere` doesn't work here for some reason.